bin/administration

```diff
@@ -124,7 +153,7 @@ class Administration
     {
         echo <<<'EOT'
 Usage:
-  administration [--delete <paste id> | --empty-dirs | --help | --purge | --statistics]
+	administration [--delete <paste id> | --empty-dirs | --help | --purge | --statistics | --list-ids]
```

Comment:
Please, replace the tabs with spaces, do also document your second option and keep the order alphabetical:
```suggestion
  administration [--delete <paste id> | --delete-all | --empty-dirs | --help | --list-ids | --purge | --statistics]
```